git_clone_test/load_project_test.ipynb

```diff
@@ -1,9 +1,22 @@
 {
```

Comment:
<p>please do not use repos owned by your private github user GiladShapira94, could be repo in some generic org like iguazio or mlrun instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>